threads/serializers.py

```
@@ -40,17 +40,59 @@ def to_representation(self, instance):
+
+
 class CommentSerializer(serializers.ModelSerializer):
+    content = serializers.CharField(required=False)
+    image = serializers.PrimaryKeyRelatedField(queryset=Image.objects.all(), required=False)
     class Meta:
         model = Comment
         fields = '__all__'
+        read_only_fields = ['user', 'thread']
+
+    def validate(self, attrs):
+        type=attrs.get("comment_type")
```

Flake8 (7.2.0)

[error] missing whitespace around operator

(E225)